UserInterface/MenuItemDemos/MenuItemDemos/MenuItemDemos/MenuItemCodePage.cs

```diff
@@ -0,0 +1,79 @@
+﻿using MenuItemDemos.Services;
+using System;
+
+using Xamarin.Forms;
+
+namespace MenuItemDemos
+{
+    public class MenuItemCodePage : ContentPage
+    {
+        Label messageLabel;
+
+        public MenuItemCodePage()
+        {
+            Padding = 10;
+            Title = "MenuItem Code Demo";
+
+            messageLabel = new Label
+            {
+                TextColor = Color.Red,
+                HorizontalOptions = LayoutOptions.CenterAndExpand,
+                VerticalOptions = LayoutOptions.Start
+            };
+
+            DataTemplate dataTemplate = new DataTemplate(() =>
+            {
+                Label label = new Label();
+                label.SetBinding(Label.TextProperty, ".");
+                ViewCell viewCell = new ViewCell
+                {
+                    View = label
+                };
+
+                MenuItem menuItem1 = new MenuItem
+                {
+                    Text = "Edit",
+                    IconImageSource = ImageSource.FromFile("icon.png")
```

Comment:
Wondering what this is for? I guess it's demoing that you can set this property. But it's quite non-intuitive that the Xamarin logo is an edit button on Android.